profile/README.md

```diff
@@ -0,0 +1,55 @@
+## Hi there, we are Fobnail Team 👋
+
+The Fobnail project provides the resources to create an axiomatically
+trustworthy device and a simple user interface to confirm the status of the
+platform. We aim to provide a reference architecture for building
+offline integrity measurement verifiers on a USB device and validators running
+in dynamically run measured environments (DLME).
+
+Our goal is to create a scalable, flexible implementation that will be the main
+building block of future (community) security solutions based on DRTM payloads.
+The owner of Fobnail can verify the reliability of the working system before
+performing any sensitive operation. Fobnail does not need an internet
+connection, which makes it immune to network stack attacks and remote
+infrastructure.
+
+[The Fobnail Token](https://fobnail.3mdeb.com/fobnail_token/) is a small
+open-source USB hardware reference project that allows a
+user/administrator/company to determine system integrity.
+
+The Fobnail project has the potential to scale up in the future. The Fobnail
+architecture can be adapted to devices using other buses than USB.
+A detailed description of the project can be found
+[here](https://fobnail.3mdeb.com/).
+
+Fobnail is a product developed by [3mdeb](https://github.com/3mdeb).
+
+## Community importance
+
+We believe that a friendly community and cooperation are key to building
+open-source trustworthy solutions. Because of that, we would like to invite you
+to:
+
+* [Fobnail Matrix Community](https://matrix.to/#/#dasharo:matrix.org) - where we
+  want to discuss Fobnail, open-source projects and another security mechanism.
+
+## Our contribution
+
+We maintain the main repositories related to the Fobnail project:
+
+* [Fobnail](https://github.com/fobnail/fobnail)
+* [Fobnail Attester](https://github.com/fobnail/fobnail-attester)
+* [Fobnail Platform Owner](https://github.com/fobnail/fobnail-platform-owner)
+
+We also maintain our forks and open-source projects like:
+
+* [cortex-mpu](https://github.com/fobnail/cortex-mpu)
+* [littlefs2-sys](https://github.com/fobnail/littlefs2-sys)
+* [littlefs2](https://github.com/fobnail/littlefs2)
+* [nrf-hal](https://github.com/fobnail/nrf-hal)
+* [lru_time_cache](https://github.com/fobnail/lru_time_cache)
+* [coap-lite](https://github.com/fobnail/coap-lite)
+* [coap-server-rs](https://github.com/fobnail/coap-server-rs)
```

Comment:
@arturkow2000 are all of this still used?

Reply:
@krystian-hebel Yes, all of them are still used